.github/workflows/build_and_test.yml

```diff
@@ -117,6 +130,19 @@ jobs:
           cd Tests/Pcap++Test
           if [ -n "$(grep avx512 /proc/cpuinfo)" ]; then Bin/Pcap++Test -n; else echo AVX-512 SUPPORT NOT FOUND, CANNOT RUN Pcap++Test; fi
 
+      - name: Create Cobertura Report
```

Comment:
The coverage report is a bit misleading for DPDK and PF_RING because we don't run these tests in CI, hence the coverage shows 0%. The reason is they require special hardware / kernel configuration which is not easy to achieve on CI machines. Maybe we can exclude these files from the report?